ngmixer/files.py

```diff
@@ -99,7 +99,6 @@ def get_meds_files(medsconf, tile_id, bands):
     fnames = []
     for band in bands:
         fname=desmeds.files.get_meds_file(medsconf, tile_id, band)
-        fname = os.path.join(dir, fname)
```

Comment:
Why did you remove this line?

Reply:
Sorry, should have commented - it causes an error in python 3:
```
 0:   File "/global/project/projectdirs/des/maccrann/py3/lib/python3.6/site-packages/ngmixer/files.py", line 102, in get_meds_files
 0:     fname = os.path.join(dir, fname)
 0:   File "/global/project/projectdirs/des/maccrann/py3/lib/python3.6/posixpath.py", line 78, in join
 0:     a = os.fspath(a)
 0: TypeError: expected str, bytes or os.PathLike object, not builtin_function_or_method
```
and anyway I assumed it was a bug since `dir` is not defined?

Reply:
it is a bug, it was probably fixed in a later update

Reply:
Hmmm. We should backport the fix maybe or just shipit. I don’t care.